style.css

```diff
@@ -78,8 +95,271 @@ h1 {
 
 .image-down {
   position: absolute;
-  width: 320px;
-  height: 160px;
+  width: 375px;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
   left: 0;
   top: 540px;
 }
+
+.projects {
+  display: grid;
+}
+
+.component {
+  /* Rectangle 21 */
+
+  position: absolute;
+  width: 375px;
+  left: 0;
+  margin-top: -274px;
+  border: 0.3333px solid rgb(255 255 255 / 10%);
+}
+
+.box {
+  /* Rectangle 22 */
+  position: absolute;
+  width: 375px;
+  height: 190px;
+  left: -1px;
+  background: #1a2236;
+  border: 0 solid rgb(255 255 255 / 10%);
+  backdrop-filter: blur(0.5px);
+
+  /* Note: backdrop-filter has minimal browser support */
+}
```

Comment:
- Kindly, use flexbox for styling your cards as your previous code reviewer suggested.👌👌